### ISSUE TITLE
F-Droid can't build - AAPT2 error

### ISSUE THREAD

**Issue by licaon-kter** [mkulesh/microMathematics#129]
```
...
> Task :app:processReleaseMainManifest
> Task :app:mergeReleaseResources FAILED
> Task :app:processReleaseManifest

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeReleaseResources'.
> Multiple task action failures occurred:
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #0: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/drawable-nodpi/shortcut_new_document.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #3: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/drawable-nodpi/app_header_teal_deep_orange.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #4: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/mipmap-xxhdpi/ic_launcher_adapt.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #2: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/mipmap-hdpi/ic_launcher_adapt.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #1: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/mipmap-xxhdpi/ic_launcher.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #5: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/mipmap-xhdpi/ic_launcher_adapt.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #7: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-4.1.0-6503028-linux Daemon #6: Unexpected error during compile '/home/vagrant/build/com.mkulesh.micromath.plus/app/src/main/res/drawable-nodpi/app_header_dark_cyan.png', attempting to stop daemon.
        This should not happen under normal circumstances, please file an issue if it does.
```

https://issuetracker.google.com/issues/184872412 This is an issue of AGP 4.1. Updating AGP to 4.2 (and maybe gradle to 7.4) or above should fix it. If that's problematic somehow, downgrading to AGP 4.0.2 works too.

/LE: fyi, https://gitlab.com/fdroid/fdroiddata/-/commit/df434c07496fe3c9bbe58b24c1b036ecf8a1c23c

**Comment by mkulesh** [comment]
Hi @licaon-kter!
thank you for the issue. I increased the AGP version to 4.2 and will submit a new release soon.

**Comment by mkulesh** [comment]
Released in version 2.23.1
